Client declares a package twice

> [ERROR] Exit code: 1 - /home/jenkins/agent/workspace/lyo-monorepo/client/oslc-java-client/src/main/java/org/eclipse/lyo/client/oslc/resources/package-info.java:15: warning: a package-info.java file has already been seen for package org.eclipse.lyo.client.oslc.resources

If I recall correctly, I did this on purpose so that oslc4j.resources.Query* stays in the old client and oslc4j.resources.*Resource is extracted into a separate package. But I guess the problem is only the javadoc:

> [ERROR] /home/jenkins/agent/workspace/lyo-monorepo/client/oslc-java-client/src/main/java/org/eclipse/lyo/client/oslc/resources/package-info.java:14: error: package org.eclipse.lyo.client.oslc.resources has already been annotated
[ERROR] @Deprecated
[ERROR] ^
[ERROR] java.lang.AssertionError

--- a/client/oslc-java-client/src/main/java/org/eclipse/lyo/client/oslc/resources/package-info.java
+++ b/client/oslc-java-client/src/main/java/org/eclipse/lyo/client/oslc/resources/package-info.java
@@ -11,5 +11,4 @@
  *
  * SPDX-License-Identifier: EPL-1.0 OR BSD-3-Clause
  */
-@Deprecated
 package org.eclipse.lyo.client.oslc.resources;

